sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/HDFSMetadataLogSuite.scala

```diff
@@ -209,14 +209,20 @@ class HDFSMetadataLogSuite extends SparkFunSuite with SharedSQLContext {
     }
 
     // Open and delete
-    val f1 = fm.open(path)
-    fm.delete(path)
-    assert(!fm.exists(path))
-    intercept[IOException] {
-      fm.open(path)
+    if (Utils.isWindows) {
+      fm.open(path).close()
+      fm.delete(path)
+      fm.delete(path) // should not throw exception
```

Comment:
Why does windows need two deletes? It seems funny we'd need an entirely different path to delete the file just here and just on Windows?